src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+
+@Entity
 public class Coach extends Person {
     public enum Track {
         JAVA, FE, UI, QA
```

Comment:
This enum will be mapped to table as numbers (0-N) by the index of element in the enum. (default hibernate behaviour)
Probably best to change this to be mapped by String representation (@Enumerated annotation)